forte/data/data_store.py

```diff
@@ -118,7 +119,6 @@ def __init__(self, onto_file_path: Optional[str] = None):
         """
         super().__init__()
         self.onto_file_path = onto_file_path
-        self.__entry_type_idx = 3
 
         """
                 The `_type_attributes` is a private dictionary that provides
```

Comment:
There is a doc spelling issue in the comment, let's fix those. Like `ints` -> `integers`